### PR TITLE
Added solar panels to the roof of LMOE bunkers

### PR DESCRIPTION
**Pull request by ActualDio**
#### Summary
None

#### Purpose of change
It doesn't make much sense that a fully independent bunker wouldn't have at least one solar panel, and rely exclusively on a gas generator.

#### Describe the solution
Simply added a new entry for a solar panel in the bunker_ext json pallete and then put four of them at the center of the lmoe roof grid

#### Describe alternatives you've considered
None

### CODE REVIEW

**Review by Maleclypse — CHANGES_REQUESTED** [review]
This would be better as a nest that has a chance to spawn rather than every single LMOE no coming with 4 solar panels, especially since solar panels would make an LMOE easier to find.

**Review by Maleclypse — APPROVED** [review]

File: data/json/mapgen/nested/lmoe_nested.json [hunk 1]
```diff
@@ -84,5 +84,15 @@
         }
       ]
     }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "This is done to give a chance for lmoe bunkers to spawn with solar panels on the roof",
+    "nested_mapgen_id": "lmoe_solar",
+    "object": { "mapgensize": [ 2, 2 ], "rows": [
```
Comment (Maleclypse) [inline data/json/mapgen/nested/lmoe_nested.json:93]: You'll need to define `M` here though

Comment (Maleclypse) [reply data/json/mapgen/nested/lmoe_nested.json:93]: @Procyonae You've been working with nested mapgen a lot lately.  Nests can't use palettes right? Or am I remembering incorrectly?

Comment (Procyonae) [reply data/json/mapgen/nested/lmoe_nested.json:93]: Nests can use palettes just fine

File: data/json/mapgen/nested/lmoe_nested.json [hunk 2]
```diff
@@ -84,5 +84,15 @@
...
+        "MM",
+        "MM"
+      ], "palettes": "roof_pallete" }
```
Comment (Procyonae) [inline data/json/mapgen/nested/lmoe_nested.json:96]:
```suggestion
      ],
      "palettes": [ "roof_pallete" ]
    }
```
Palettes has to be an array

**Pull request merged**